pandas/core/generic.py

```diff
@@ -5716,24 +5716,86 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Return copy of the input with values below given value(s) trimmed.
+
+        Elements below the `threshold` will be changed to match the
+        `threshold` value(s).
 
         Parameters
         ----------
-        threshold : float or array_like
-        axis : int or string axis name, optional
+        threshold : float or array-like
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
+
+            .. versionadded:: 0.21.0
 
         See Also
         --------
-        clip
+        DataFrame.clip : General purpose method to trim `DataFrame` values to
+            given threshold(s)
+        DataFrame.clip_upper : Trim `DataFrame` values above given
+            threshold(s)
+        Series.clip : General purpose method to trim `Series` values to given
+            threshold(s)
+        Series.clip_lower : Trim `Series` values below given threshold(s)
+        Series.clip_upper : Trim `Series` values above given threshold(s)
 
         Returns
         -------
-        clipped : same type as input
+        Series or DataFrame
+            Same type as caller.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'a': [0.740518, 0.450228, 0.710404, -0.771225],
+        ...                    'b': [0.040507, -0.45121, 0.760925, 0.010624]})
+
+        >>> df
+                  a         b
+        0  0.740518  0.040507
+        1  0.450228 -0.451210
+        2  0.710404  0.760925
+        3 -0.771225  0.010624
+
+        Clip to a scalar value
+
+        >>> df.clip_lower(0.2)
+                  a         b
+        0  0.740518  0.200000
+        1  0.450228  0.200000
+        2  0.710404  0.760925
+        3  0.200000  0.200000
+
+        Clip to an array along the index axis
+
+        >>> df.clip_lower([0.2, 0.4, 0.6, 0.8], axis=0)
+                  a         b
+        0  0.740518  0.200000
+        1  0.450228  0.400000
+        2  0.710404  0.760925
+        3  0.800000  0.800000
+
+        Clip to an array column the index axis
```

Comment:
I believe for consistency here you meant to say "Clip to an array along the column axis"